src/CL/Slack/Resources/doc/methods/search-all.md

```diff
@@ -26,7 +26,7 @@ $response = $apiClient->send($payload);
 
 if ($response->isOk()) {
     // query has been executed and result is returned (but can be empty)
-    $response->getMessageResult()->getMatches(); // Message objects
+    $response->getResult()->getMatches(); // Message objects
```

Comment:
Can you explain this, the `SearchAllPayloadResponse` class doesn't have a `getResult` method